plugin/src/test/java/io/jenkins/plugins/casc/SampleBenchmark.java

```diff
@@ -0,0 +1,28 @@
+package io.jenkins.plugins.casc;
+
+import io.jenkins.plugins.casc.misc.jmh.CascJmhBenchmarkState;
+import java.io.IOException;
+import javax.annotation.Nonnull;
+import jenkins.benchmark.jmh.JmhBenchmark;
+import jenkins.model.Jenkins;
+import org.openjdk.jmh.annotations.Benchmark;
+
+import static org.junit.Assert.assertEquals;
+
+@JmhBenchmark
+public class SampleBenchmark {
+    public static class MyState extends CascJmhBenchmarkState {
+        @Nonnull
+        @Override
+        protected String getResourcePath() {
+            return "io/jenkins/plugins/casc/benchmarks.yml";
```

Comment:
shouldn't need the full resource path as the current package would be looked in first?
```suggestion
            return "benchmarks.yml";
```

Reply:
@timja It doesn't work without the providing full path to the file. I get a `null` otherwise.

Reply:
I would like to see this, see my other review comment